exercises/concept/cars-assemble/src/test/java/CarsAssembleTest.java

```diff
@@ -9,67 +11,91 @@ public class CarsAssembleTest {
     private CarsAssemble carsAssemble;
     private double epsilon = 0.0000001d;
 
-    @Before
+    @BeforeEach
     public void setUp() {
         carsAssemble = new CarsAssemble();
     }
 
     @Test
+    @Tag("task:1")
+    @DisplayName("The productionRatePerHour method returns the correct result given line's speed")
```

Comment:
The test runner requires that each test has a unique name. Plus, for students it’s also more meaningful if each test is clearly named.

I would suggest naming these methods “The productionRatePerHour method returns the correct result when the line’s speed is 0/1/etc”. Or maybe even “The productionRatePerHour should be X for line speed Y”.

Reply:
Very clear, I will keep it in mind for next PRs, thanks!